### ISSUE TITLE
ADBMobile: Enable bitcode for arm64

### ISSUE THREAD

**Issue by AdrienMgm** [Adobe-Marketing-Cloud/mobile-services#309]
Hello,

When I try to build my project with the enable bitcode build option to yes I got this error message from Xcode:

`AdobeMobileLibrary.a(ADBMobile.o)' does not contain bitcode. You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target. file 'Builds/iOS/Libraries/Plugins/iOS/AdobeMobileLibrary.a' for architecture arm64`

Could you please provide an updated library with this option activated?

**Comment by sbenedicadb** [comment]
Hey @AdrienMgm,

Would you mind sharing which version of the SDK you are using?  Also, any other details you can share about your environment would be helpful.  

- What version of Xcode are you using?
- Is your app Swift or Objective-c?
- Are you doing a manual installation or are you using Cocoapods?
- What iOS version does your app target?
- What architectures does your app support?

Thanks.

**Steve Benedick**
_Sr. Mobile Engineer_
[Adobe Experience Cloud](https://forums.adobe.com/external-link.jspa?url=https%3A%2F%2Fwww.adobe.com%2Fexperience-cloud.html)

**Comment by AdrienMgm** [comment]
Hey @sbenedicadb,

We are using Unity for an iOS game so the last available version is [4.13.1 (20 Oct, 2016)](https://github.com/Adobe-Marketing-Cloud/mobile-services/releases/tag/v4.13.1-Unity).
I wanted to update the iOS library by just replacing the files in plugins/iOS folder hoping that the bitcode option will be available.

> What version of Xcode are you using?

The last one 10.2

> Is your app Swift or Objective-c?

Unity stack (C#, il2cpp compiler, Objective-c plugins)

> Are you doing a manual installation or are you using Cocoapods?

I used unity package. No Cocoapods. Also I had to add libsqllite3 before compiling with xcode.

> What iOS version does your app target?

From 11.0 to the latest.

> What architectures does your app support?

Only ARM64

Thanks.

**Adrien M.**
